src/olgm/gm.js

```diff
@@ -83,36 +85,39 @@ olgm.gm.createLatLng = function(object, opt_ol3map) {
 
 /**
  * Create a Google Maps LineString or Polygon object using an OpenLayers one.
- * @param {ol.geom.LineString|ol.geom.Polygon} geometry geometry to create
+ * @param {ol.geom.LineString|ol.geom.MultiLineString|ol.geom.Polygon|ol.geom.MultiPolygon} geometry geometry to create
  * @param {ol.Map=} opt_ol3map For reprojection purpose. If undefined, then
  *     `EPSG:3857` is used.
- * @return {google.maps.Data.LineString|google.maps.Data.Polygon} google
- * LineString or Polygon
+ * @return {google.maps.Data.LineString|google.maps.Data.MultiLineString|google.maps.Data.Polygon} google
+ * LineString, MultiLineString or Polygon
  */
 olgm.gm.createGeometry = function(geometry, opt_ol3map) {
   var inProj = (opt_ol3map !== undefined) ?
       opt_ol3map.getView().getProjection() : 'EPSG:3857';
 
-  var latLngs = [];
-  var lonLatCoords;
-
-  var coordinates;
-  if (geometry instanceof ol.geom.LineString) {
-    coordinates = geometry.getCoordinates();
-  } else {
-    coordinates = geometry.getCoordinates()[0];
-  }
-
-  for (var i = 0, len = coordinates.length; i < len; i++) {
-    lonLatCoords = ol.proj.transform(coordinates[i], inProj, 'EPSG:4326');
-    latLngs.push(new google.maps.LatLng(lonLatCoords[1], lonLatCoords[0]));
-  }
+  var genLatLngs = function(coordinates) {
+    var latLngs = [];
+    var lonLatCoords;
+    for (var i = 0, len = coordinates.length; i < len; i++) {
+      lonLatCoords = ol.proj.transform(coordinates[i], inProj, 'EPSG:4326');
+      latLngs.push(new google.maps.LatLng(lonLatCoords[1], lonLatCoords[0]));
+    }
+    return latLngs;
+  };
```

Comment:
Move this method out, i.e. something like `olgm.gm.coordinatesToLatLngs`.

Reply:
Done